skit.API/Controllers/Areas/Auth/AccountController.cs

```diff
@@ -11,11 +11,11 @@ public sealed class AccountController : BaseController
     [HttpPost("sign-up-company")]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
-    public async Task<IActionResult> SignUpCompany([FromBody] SignUpCompanyCommand command,
+    public async Task<ActionResult<JwtWebToken>> SignUpCompany([FromBody] SignUpCompanyCommand command,
         CancellationToken cancellationToken)
```

Comment:
Czy powinno się bez typu generycznego używać ActionResult? Czyli czy w linii 14 wcześniej mogło być po prostu ActionResult zamiast IActionResult?

Reply:
Ja zawsze w projektach widziałem coś takiego: jak nic nie jest zwracane to IActionResult, jak zwracamy jakieś dane to ActionResult. Ale czemu tak to nie wiem XD